include/command-line-processing.php

```diff
@@ -70,7 +70,7 @@
 </ol>
 
 <p>You can find a detailed explanation of each of the constituent parts of the command-line in the sections that follow.</p>
-
+'
```

Comment:
Thanks for the h5 fixes but why was this added?

Reply:
probably a typo from me :-)